interrupt-handlers.go

```diff
@@ -0,0 +1,13 @@
+package cli
+
+import "os"
+
+var ExitOnInterrupt InterruptHandlerFunc = func(ctx *Context) {
+	CancelContextOnInterrupt(ctx)
+	<-ctx.Done()
+	os.Exit(1)
```

Comment:
I don't think we should cancel the context in this case.

Reply:
To be honest, if we start passing down signals, I think default should be os.Exit on sigterm, cancel context on sigint